docs/indicator_creation.md

```diff
@@ -106,3 +106,16 @@ Finally, you need to create a svg figure (e.g. with matplotlib) and save it to `
 If you have defined these three functions, your indicator is ready to go. To show how OQT uses your indicator to be displayed on the OQT Website or in your command line interface, we made a sequence diagram. 
 
 ![UML Sequence Diagram](img/UML-Sequence-Diagram.png)
+
+
+#### A note on using `scikit-learn` models
```

Comment:
This can be extended on once we implement our first Indicator using this feature.